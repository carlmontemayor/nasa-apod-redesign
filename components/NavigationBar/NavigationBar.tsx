import { ReactElement } from 'react'

// This is a demo of husky, a git tool.
export const NavigationBar = (): ReactElement => {
  return (
    <nav>
      <div className="flex relative">
        <div className="font-bold text-5xl p-4 pr-0">APOD</div>
        <div className="font-bold text-sm center pr-4 pt-10 ">
          (astronomy picture of the day)
        </div>
      </div>
      <div className="font-bold text-sm  bottom-0 right-0 absolute p-2 ">
        Astronomy Picture of the Day
      </div>
    </nav>
  )
}
