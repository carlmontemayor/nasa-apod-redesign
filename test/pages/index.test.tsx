import React from 'react'
import { render } from '../testUtils'
import { NavigationBar } from '../../components/NavigationBar/NavigationBar'

describe('Home page', () => {
  it('matches snapshot', () => {
    const { asFragment } = render(<NavigationBar />, {})
    expect(asFragment()).toMatchSnapshot()
  })
})
