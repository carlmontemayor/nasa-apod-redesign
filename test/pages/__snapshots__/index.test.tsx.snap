// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home page matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="bg-black"
  >
    this is tailwind.
  </div>
</DocumentFragment>
`;
